code/game/objects/items/explosives/mine.dm

```diff
@@ -286,3 +286,162 @@
 	customizable = TRUE
 	matter = list("metal" = 3750)
 	has_blast_wave_dampener = TRUE
+
+
+
+/obj/item/explosive/mine/bounding
+	name = "\improper M5 Bounding Mine"
+	desc = "The M5 Bounding Mine is a landmine that propels itself upwards when the tripwire is set off. It explodes mid-air and either maims, kills, or fucks up anything standing in the blast zone. The 'Bouncing Betty' has seen limited use since the year 2150 due to the IFF sensors on the earlier models were prone to failure, resulting in friendly casualties. Be careful to not trip this or you're going to be ending up in a hospital bed full of shrapnel."
+	icon = 'icons/obj/items/weapons/grenade.dmi'
+	icon_state = "m5"
+	force = 5.0
+	w_class = SIZE_SMALL
+	// layer = MOB_HIDING_LAYER = 0.1 // geeves told me this was a good idea but it didnt work so it's gonna be like claymores lol
```

Comment:
Not sure exactly what Geeves wanted here but remove the code rather than comment it.